pavelib/paver_tests/test_prereqs.py

```diff
@@ -4,7 +4,11 @@
 
 import os
 import unittest
-from pavelib.prereqs import no_prereq_install
+from mock import call, patch
+from paver.easy import BuildFailure
+from pavelib.prereqs import no_prereq_install, node_prereqs_installation
+from pavelib.paver_tests.utils import PaverTestCase, CustomShMock
+from pavelib.paver_tests.test_paver_quality import CustomShMock
```

Comment:
I think it'd be fine to just move them out of the class and import the functions rather than the class.